cunumeric/linalg/cholesky.py

```diff
@@ -202,12 +202,37 @@ def tril(context: Context, p_output: StorePartition, n: int) -> None:
     task.execute()
 
 
+def batched_cholesky(output: DeferredArray, input: DeferredArray) -> None:
```

Comment:
Is this going to be an internal method or exposed to user? In the first case I would name it with `_` like `_batched_cholesky`

Reply:
added underscore, private method